.yarnrc.yml

```diff
@@ -0,0 +1,17 @@
+enableScripts: false
+
+enableTelemetry: 0
+
+logFilters:
+  - code: YN0004
+    level: discard
+
+nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-allow-scripts.cjs
+    spec: 'https://raw.githubusercontent.com/LavaMoat/LavaMoat/main/packages/yarn-plugin-allow-scripts/bundles/@yarnpkg/plugin-allow-scripts.js'
+  - path: .yarn/plugins/@yarnpkg/plugin-constraints.cjs
```

Comment:
I'd either copy the constraints from the module template as well, or remove this plugin.